Gemfile.jruby-1.9.lock

```diff
@@ -132,8 +132,8 @@ GEM
     logstash-codec-graphite (2.0.2)
       logstash-codec-line
       logstash-core (>= 2.0.0.beta2, < 3.0.0)
-    logstash-codec-json (2.1.0)
-      logstash-core (>= 2.0.0.beta2, < 3.0.0)
+    logstash-codec-json (2.1.1)
+      logstash-core (>= 2.0.0, < 3.0.0)
```

Comment:
It was updated in this commit in https://github.com/logstash-plugins/logstash-codec-json/commit/fd0b6efbb851ece48a37466eb10eff69485dec35 in 2.1.1